src/options.cpp

```diff
@@ -1348,8 +1348,7 @@ void register_options(void)
    unc_add_option("align_assign_thresh", UO_align_assign_thresh, AT_UNUM,
                   "The threshold for aligning on '=' in assignments (0=no limit)", "", 0, 5000);
    unc_add_option("align_enum_equ_span", UO_align_enum_equ_span, AT_UNUM,
-                  "The span for aligning on '=' in enums (0=don't align)\n"
-                  "Note: align_assign_span must be set also.", "", 0, 5000);
+                  "The span for aligning on '=' in enums (0=don't align)\n", "", 0, 5000);
```

Comment:
Github tracks changes on the branch of the PR, if you add an additional commit it will be added here.
Since we don't do often squashes here (if at all) I recommend a different approach where you fix you mistakes, and integrate them into your old commits.
The change that will be necessary here are done in the last commit so you can stage the changes and use `git commit --amend --no-edit`.
This will integrate the staged changes into you last commit, as this changes the history of your commits and you will need do a force (`-f`) push.